devel-common/src/sphinx_exts/operators_and_hooks_ref.py

```diff
@@ -481,6 +481,17 @@ def render_content(
         )
 
 
+class QueuesDirective(BaseJinjaReferenceDirective):
+    """Generate list of queues"""
+
+    def render_content(
+        self, *, tags: set[str] | None, header_separator: str = DEFAULT_HEADER_SEPARATOR
+    ) -> str:
+        return _common_render_list_content(
+            header_separator=header_separator, resource_type="queues", template="queues.rst.jinja2"
+        )
+
+
```

Comment:
I did not know we could do that! Super cool!

Reply:
Yeah. And with the new workflow `uv run ... --autobuild` things - it all runs locally with all the needed dependencies, and you get quick feedback and it will reload itself after change without restarting the build so it's WAY easier to add such code and iterate over it. I literally started to like sphinx after making this change.